python/raydp/torch/estimator.py

```diff
@@ -211,15 +215,15 @@ def train_func(config):
         metrics = config["metrics"]
 
         # create dataset
-        train_data_shard = get_dataset_shard("train")
+        train_data_shard = session.get_dataset_shard("train")
```

Comment:
The latest Ray document suggests using `iter_torch_batches` instead of `to_torch` on the data shard. Can you please take a look at the difference?